README.md

```diff
@@ -276,6 +276,25 @@ file, rename and edit it.
 <?php
 
 $settings = array (
+    // Security settings that must be set to avoid insecure SHA-1 usage.
```

Comment:
Basic settings only contain IdP and SP info, security stuff is on advanced_settings

If you want you can add a new section aboutthe sha1 deprecation info on the README after
https://github.com/onelogin/php-saml#idp-with-multiple-certificates

Something like  SHA-1 Deprecation, and explain it why is important